### DISCUSSION TITLE
How to specify fractional time duration?

### DISCUSSION THREAD

**Discussion by gdevenyi** [nextflow-io/nextflow#4756] [Q&A]
In the code definition of duration a period (`.`) is used to separate the numeric from the units, however, this causes a problem when I want to specify a fractional unit (`1.5.hr`). I get an error when doing so in the config/profile settings.

How do we specify fractional durations with units attached?

**Comment by bentsherman** [comment]
You can combine different time units: https://nextflow.io/docs/latest/process.html#time

**Reply by gdevenyi** [reply]
That example is from "string" formats, but in profile/process config it looks like I need the numeric representation.

I can't find an example for that

**Reply by bentsherman** [reply]
You can use the string representation in config files too
